release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -180,8 +180,8 @@ In {product-title} 4.12, cluster deployments to {rh-openstack-first} clouds that
 For a brief overview of this type of deployment, see the blog post link:https://cloud.redhat.com/blog/deploying-your-cluster-at-the-edge-with-openstack[Deploying Your Cluster at the Edge With OpenStack].
 
 [id="ocp-4-12-aws-outposts"]
-==== {product-title} on AWS Outposts
-{product-title} {product-version} is now supported on the AWS Outposts platform. With AWS Outposts you can deploy edge-based worker nodes, while using AWS Regions for the control plane nodes. The documentation for this feature is currently unavailable and is targeted for release at a later date.
+==== {product-title} on AWS Outposts (Technology Preview)
+{product-title} {product-version} is now supported on the AWS Outposts platform as a link:https://access.redhat.com/support/offerings/techpreview[Technology Preview]. With AWS Outposts you can deploy edge-based worker nodes, while using AWS Regions for the control plane nodes. For more information, see xref:../installing/installing_aws/installing-aws-outposts-remote-workers.adoc[Installing a cluster on AWS with remote workers on AWS Outposts].
```

Comment:
Should there be a comma in this sentence?  "...can deploy edge-based worker nodes, while using..."

Reply:
Agree. No comma helps readability + there's no reason for it

Reply:
@eohartman @stevsmit Thanks for taking the time to review. I will apply this suggestion.